sync/config/results.yaml

```diff
@@ -0,0 +1,45 @@
+# Each YAML file under sync/ configures how sync/sync.py synchronizes
+# contents of various versions from its source of truth (usually a GitHub
+# repository of a Tekton component, such as tektoncd/pipelines) to
+# content/ (for the lastest version) and vault/ (for earlier versions).
+
+# The name of the component.
+# sync.py will use this value to build directories in content/ and vault/.
+component: Results
+# The GitHub repository where documentation resides.
+repository: https://github.com/tektoncd/results
+# The link to the GitHub tag page.
+archive: https://github.com/tektoncd/results/tags 
+# The tags (versions) of contents to sync.
+# Note that sync.py and related script reads tags in the order specified in
+# the following list; the first entry in tags will automatically become the
+# latest version of contents.
+# To add a new version, append to the list as below
+# - name: vX.Y.Z
+#   displayName: vX.Y.Z
+#   folders:
+#     docs:
+#       target: '/'            # optional, default value '/'
+#       index: README.md       # optional, if _index.md exists
+#       include: ['*.md']      # optional, default value ['*']
+#       exclude: ['not_in.md'] # optional, default value []
+#       header: <dict>         # optional, no header added if not set
+#         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
+tags:
+- name: v0.3.1
```

Comment:
This works, however it won't let you pull in any docs fixes (like the [front matter](https://github.com/tektoncd/results/pull/144) one) unless you create a new tag or update this one.
I would recommend creating a release-v0.3.x branch and configure it here, so that you can then push docs fixes and they'll be automatically be picked up during the nightly deploy (if not earlier). 

You could also include docs from the `main` branch, which is what all other projects do.

Reply:
Done. Using main for now.